pkg/depository/create.go

```diff
@@ -203,3 +208,52 @@ func getNonce(h string, account string) uint64 {
 
 	return n.Nonce
 }
+
+func getWalletInfo(walletDir string, account string) (common.WalletConfig, error) {
+	obj := common.WalletConfig{
+		Address:    "",
+		PrivateKey: nil,
+	}
+
+	if walletDir == "" {
+		walletDir = common.WalletConfigDir
+	}
+	walletDir = strings.TrimSuffix(walletDir, "/")
+	_, err := os.Stat(walletDir)
+	if err != nil {
+		return obj, fmt.Errorf("walletDir error: %s", err)
+	}
+
+	// get accounts
+	dirEntries, err := os.ReadDir(walletDir)
+	if err != nil {
+		return obj, fmt.Errorf("get dirEntries error: %s", err)
+	}
+
+	var hasAccount bool
+
+	// read account info
+	for _, info := range dirEntries {
+		if info.IsDir() {
```

Comment:
The account name is also the name of the file, so you can read the file directly without traversing the directory.